doc/specifications/proposed/named_queries/named_queries.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+Named queries are Content or Location Query objects that are predefined in the system.
+
+They can be used in multiple ways:
+- from twig templates, using a sub-request: {{ render( controller( ez_query, {'identifier': 'latest_articles' ) ) }}
+- from php code, by getting the Query from a service
+- from REST, by executing a predefined view / query
+
+They
+## Components
+
+### REST
+
+All REST calls on `/content/views`.
+
+### Query controller
+```twig
+{{ render( controller( ez_query, {'identifier': 'latest_articles', 'parameters': {'type': 'article'}} ) ) }}
+```
+
+### Services
+
+A main `QueryService` / `ezplatform.query_service`, with methods to load/create/update/delete named queries.
```

Comment:
AFAIR of our discussions, they're no different in essence. 
One is defined as a *type* (like a FormType), the other is persisted. In both cases, the result of the executed query is a *view*.